### ISSUE TITLE
[Spec Contradiction] NationEnvironment action space violates 07_AGENT_ACTION_SPACE.md

### ISSUE THREAD

**Issue by thewildofficial** [Kr0issant/nation-optimizer-rl#10]
## Problem
The `NationEnvironment` wrapper uses a continuous 6-dimensional bid vector (`NationAction(bids=[...])`) mapped through softmax to direct allocations. This completely bypasses the structured parliamentary action space defined in `specification/07_AGENT_ACTION_SPACE.md` and the adapter contract in `specification/13_RL_ADAPTERS_AND_TRAINING.md`.

## Current `NationAction`
```python
class NationAction(BaseModel):
    bids: list[float]  # length 6, softmax -> allocation percentages
```

## Required per spec (`07_AGENT_ACTION_SPACE.md`)
- `PROPOSE_BUDGET` (department, amount, justification)
- `VOTE` (proposal_id, vote)
- `DEBATE` (message)
- `ABSTAIN_FROM_PROPOSAL`

## Impact
- The wrapper is unsuitable for training parliamentary LLM adapters or dictator LLM adapters because it eliminates the proposal, voting, and debate phases.
- `schemas/actions.py` defines the structured action schema, but `server/models.py` ignores it.
- This contradicts the core game loop in `core/game.py` which implements the full 9-phase parliamentary cycle.

## Suggested resolution
Either:
1. Redesign `NationEnvironment` to accept and emit the structured parliamentary action/observation schema (complex but spec-compliant), OR
2. Explicitly document the continuous-bid wrapper as a simplified "direct allocation" training mode, separate from the parliamentary mode.
